hoomd/hpmc/validation/lj_spheres.py

```diff
@@ -109,7 +109,10 @@ def accumulate_energy(timestep):
             mc.set_params(d=0, a=0); # test cluster moves alone
 
         # Sample
-        run(1000,callback=accumulate_energy, callback_period=10)
+        if use_clusters:
+            run(5000,callback=accumulate_energy, callback_period=10)
```

Comment:
Do these old `run` calls still work? This is no longer the syntax used for version 3 simulations.

Reply:
No, they won't work. We need to port over the validation tests from 2 to 3 in a separate PR.